floris/utilities.py

```diff
@@ -168,6 +177,41 @@ def tand(angle):
     return np.tan(np.radians(angle))
 
 
+# @jit(nopython=True)
```

Comment:
Do we want to keep these two functions (`wrap_180` and `wrap_180_fast`) if we aren't doing jit?

Reply:
I removed them, good catch